### PR TITLE
Created a function to reset Account using firebase

### PR DESCRIPTION
**Pull request by OladipoOmotosho**
Created a function to reset Account using firebase

### CODE REVIEW

**Review by Remi-dee — CHANGES_REQUESTED** [review]
Hi @OladipoOmotosho, kindly turn your function to named export

File: src/composables/resetAccountWithFirebase.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { updatePassword } from "firebase/auth";
+import { appAuth } from "./firebaseConfig/config";
+
+export function resetUserAccount(email, newPassword) {
+    const auth = appAuth();
```
Comment (annonymousauthority) [inline src/composables/resetAccountWithFirebase.js:5]: No need for this

File: src/composables/resetAccountWithFirebase.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    auth.sendPasswordResetEmail(email)
```
Comment (annonymousauthority) [inline src/composables/resetAccountWithFirebase.js:7]: `sendPasswordResetEmail` is not a method to auth... Use the documentation to see appAuth should be a parameter just like email

File: src/composables/resetAccountWithFirebase.js
Comment (annonymousauthority) [inline src/composables/resetAccountWithFirebase.js:1]: remove all console.log error

Use a try-Catch block so you can return a successful boolean value either `true` or `false`

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

**Review by annonymousauthority — APPROVED** [review]
LGTM - Update on the trello board as well

**Review by Remi-dee — APPROVED** [review]

**Pull request merged**